### DISCUSSION TITLE
Basic_Accounting_w_nat_resources

### DISCUSSION THREAD

**Discussion by MacroPY** [julicaunedo/STEG_Lecture2#2] [General]
Sorry for the inconvenience, but honestly I am not very good at stata, and I would like to know if anyone else had any problems when calculating the capital output ratio, specifically at the beginning with g capital_output_ratio=ln(cn/cgdpo)*`alpha'/(1-`alpha').

By the way, the version of stata I have is 15.1.

Best regards,

W.G.

**Comment by julicaunedo** [comment]
Hi W.G. I think what is going on is that if you try to run the line as a stand-along (from the command line) stata will spit an error because that line uses a local variable `alpha' which needs to be defined.  You should free to ask more during TA session on Monday.

**Reply by MacroPY** [reply]
Hi Julieta, thank you very much for the answer, in any case I also tried what you mentioned, it didn't work either, which I found quite strange. And yes, I will try to ask that on Monday.

In any case, thank you very much for the class, sincerely it was very good, and I enjoyed it a lot,

Best regards

W.G.
